app/code/Magento/Checkout/Test/Unit/Block/Cart/ShippingTest.php

```diff
@@ -3,122 +3,215 @@
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var LayoutProcessorInterface|MockObject
      */
-    protected $layoutProcessor;
+    protected $layoutProcessorMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var StoreManagerInterface|MockObject
      */
-    protected $storeManager;
+    protected $storeManagerMock;
 
     /**
      * @var array
      */
     protected $layout;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var Json|MockObject
      */
-    private $serializer;
+    private $serializerMock;
 
-    protected function setUp()
+    /**
+     * @var JsonHexTag|MockObject
+     */
+    private $jsonHexTagSerializerMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp(): void
     {
-        $this->context = $this->createMock(\Magento\Framework\View\Element\Template\Context::class);
-        $this->customerSession = $this->createMock(\Magento\Customer\Model\Session::class);
-        $this->checkoutSession = $this->createMock(\Magento\Checkout\Model\Session::class);
-        $this->configProvider = $this->createMock(\Magento\Checkout\Model\CompositeConfigProvider::class);
-        $this->layoutProcessor = $this->createMock(\Magento\Checkout\Block\Checkout\LayoutProcessorInterface::class);
+        $this->contextMock = $this->createMock(Context::class);
+        $this->customerSessionMock = $this->createMock(customerSession::class);
+        $this->checkoutSessionMock = $this->createMock(checkoutSession::class);
+        $this->configProviderMock = $this->createMock(CompositeConfigProvider::class);
+        $this->layoutProcessorMock = $this->createMock(LayoutProcessorInterface::class);
+        $this->serializerMock = $this->createMock(JsonHexTag::class);
+        $this->jsonHexTagSerializerMock = $this->createMock(JsonHexTag::class);
+        $this->storeManagerMock = $this->createMock(StoreManagerInterface::class);
         $this->layout = [
             'components' => [
-                'firstComponent' => ['param' => 'value'],
-                'secondComponent' => ['param' => 'value'],
+                'firstComponent' => ['param' => 'value']
             ]
         ];
 
-        $this->storeManager = $this->createMock(\Magento\Store\Model\StoreManagerInterface::class);
-        $this->context->expects($this->once())->method('getStoreManager')->willReturn($this->storeManager);
-        $this->serializer = $this->createMock(\Magento\Framework\Serialize\Serializer\Json::class);
+        $this->contextMock->expects($this->once())
+            ->method('getStoreManager')
+            ->willReturn($this->storeManagerMock);
 
-        $this->model = new \Magento\Checkout\Block\Cart\Shipping(
-            $this->context,
-            $this->customerSession,
-            $this->checkoutSession,
-            $this->configProvider,
-            [$this->layoutProcessor],
+        $this->block = new Shipping(
```

Comment:
Why don't you use ObjectManager helper that mocks all the required classes itself?

Reply:
is it necessary to add ObjectManager helper?

Reply:
It's highly recommended to initiate classes with ObjectManager helper.

Reply:
@lbajsarowicz Thanks for the feedback, then surely I will update.

Reply:
@lbajsarowicz I have updated the changes. please review now.

Reply:
@lbajsarowicz  I have updated the changes, please review